src/Gating/Configuration/GatingConfiguration.cs

```diff
@@ -8,14 +8,13 @@
 // </auto-generated>
 //------------------------------------------------------------------------------
 
-// 
+//
 // This source code was auto-generated by xsd, Version=4.7.2046.0.
```

Comment:
>generated [](start = 29, length = 9)

This file is autogenered, so ensure the template is also updated to ensure this doesn't revert

Reply:
There could actually be other templates that need to be modified too. Please check for .txt files like this

---
In reply to: [395092251](https://github.com/microsoft/Omex/pull/170#discussion_r395092251) [](ancestors = 395092251)

Reply:
Thank you I've forgotten about txt templates, just pushed updates for txt templates in gating